api/resources/translations/messages-en.properties

```diff
@@ -1024,6 +1024,8 @@ permission.description.can_view_unallocated_data_records = Allowed to see report
 permission.description.can_view_users = Allowed to get a list of all configured users.
 permission.description.can_write_wealth_quintiles = Allowed to update contacts with their wealth quintiles.
 permission.description.can_upgrade = Allowed to upgrade the CHT Core Framework version via the API or admin interface.
+permission.description.can_have_multiple_places.not_allowed = The person with selected role cannot have multiple places.
+permission.description.can_have_multiple_places.incompatible_place = The selected places must be at the same hierarchy level.
```

Comment:
I think this message can be misleading, because what we are actually checking is whether the selected places are of the same type, not that they are at the same hierarchy level. 
Because a certain contact_type can have multiple parent types, these two assertions are not equivalent. 
I think this message should convey what we are actually checking, that the places have the same type. 

If we want to switch our verification to check for the level, then code changes are required.

Reply:
Could we have it as `The selected places must be of the same type (be at the same hierarchy level).` ? That conveys technically what we are checking, but also explaining to the user in a way they understand.